contracts/atomic-swap/examples/schema.rs

```diff
@@ -0,0 +1,21 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema, export_schema_with_title, remove_schemas, schema_for};
+
+use atomic_swap::msg::HandleMsg;
+use atomic_swap::msg::InitMsg;
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InitMsg), &out_dir);
+    export_schema_with_title(&mut schema_for!(HandleMsg), &out_dir, "HandleMsg");
```

Comment:
This doesn't need to include variants (that are pulled in from `HandleMsg`, but should also include return values, which are not. Please add:

* `QueryMsg`
* `ListResponse`
* `DetailsResponse`